### PR TITLE
Sprint 2 Demo

### PR DESCRIPTION
**Pull request by jaureguif**

### CODE REVIEW

**Review by dan-zx — APPROVED** [review]
Looks good to me

File: .checkstyle [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (dan-zx) [inline .checkstyle:1]: This looks like an autogenerated file

**Review by dan-zx — CHANGES_REQUESTED** [review]
Except from the file .checkstyle, it looks good to me. 
I couldn't find the setting to suppress the checkstyle logs in the console

File: .checkstyle (hunk 1 above)
Comment (dioniciorodriguez) [reply .checkstyle:1]: niceee teeeaam!!

**Review by dioniciorodriguez — COMMENTED** [review]
ok

**Pull request closed without merging**